eventually/static/src/containers/fileUpload/FileUpload.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import axios from 'axios';
+import Dropzone from 'react-dropzone';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import LinearProgress from 'material-ui/LinearProgress';
+import Snackbar from 'material-ui/Snackbar';
+
+const max_image_size = 8*1024*1024;
+const possible_image_types = ['gif', 'png', 'jpg', 'jpeg', 'bmp'];
+const request_path = '/api/v1/img/handle/';
+const request_timeout = 30000;
+const dialogStyle = {'opacity': '0.98'};
```

Comment:
please, can you rewrite keys of style objects without `'`? Because we merge in dev code without them, and it might be not 'true' convention style =)

Reply:
Agreed, corrected.